### PR TITLE
Test definition and Test score definition

### PR DESCRIPTION
**Pull request by xinnanlisf**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: force-app/main/default/objects/Test_Definition__mdt/Test_Definition__mdt.object-meta.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <description>Identifies the type of Test taken and the Test provider. For example, the SAT administered by the College Board or X.</description>
```
Comment (MatthewBlanski) [inline force-app/main/default/objects/Test_Definition__mdt/Test_Definition__mdt.object-meta.xml:3]: Suggestion from docs:
```suggestion
    <description>Identifies the type of Test taken and the Test provider. For example, the SAT administered by the College Board.</description>
```

File: force-app/main/default/objects/Test_Definition__mdt/fields/Test_Type__c.field-meta.xml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    <label>Test Type</label>
+    <required>true</required>
+    <type>Picklist</type>
+    <valueSet>
+        <restricted>true</restricted>
+        <valueSetDefinition>
+            <sorted>false</sorted>
+            <value>
+                <fullName>SAT</fullName>
+                <default>true</default>
```
Comment (MatthewBlanski) [inline force-app/main/default/objects/Test_Definition__mdt/fields/Test_Type__c.field-meta.xml:17]: I don't feel like we need to have a default here.
```suggestion
                <default>false</default>
```

File: force-app/main/default/objects/Test_Definition__mdt/fields/Academic_Level__c.field-meta.xml [hunk 3]
```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Academic_Level__c</fullName>
+    <description>The grade-level, developmental level, or age-level that the Test Definition was designed for.</description>
+    <externalId>false</externalId>
+    <fieldManageability>DeveloperControlled</fieldManageability>
```
Comment (MatthewBlanski) [inline force-app/main/default/objects/Test_Definition__mdt/fields/Academic_Level__c.field-meta.xml:6]: Let's open up the fields on all of these to allow for any user to modify them. Same for test score.

File: force-app/main/default/objects/Test_Score_Definition__mdt/fields/Possible_Results__c.field-meta.xml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Possible_Results__c</fullName>
+    <description>For tests that use non-numerical scores, the possible results expressed as a comma-delimited list. For example, &quot;Pass, Fail&quot; or &quot;A&quot;, &quot;B&quot;, &quot;C&quot;, &quot;D&quot;, &quot;F&quot;.</description>
```
Comment (MatthewBlanski) [inline force-app/main/default/objects/Test_Score_Definition__mdt/fields/Possible_Results__c.field-meta.xml:4]: Every value should be wrapped in quotes here.
```suggestion
    <description>For tests that use non-numerical scores, the possible results expressed as a comma-delimited list. For example, &quot;Pass&quot;, &quot;Fail&quot; or &quot;A&quot;, &quot;B&quot;, &quot;C&quot;, &quot;D&quot;, &quot;F&quot;.</description>
```

File: force-app/main/default/objects/Test_Score_Definition__mdt/fields/Possible_Results__c.field-meta.xml [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+    <externalId>false</externalId>
+    <fieldManageability>DeveloperControlled</fieldManageability>
+    <inlineHelpText>For tests that use non-numerical scores, the possible results expressed as a comma-delimited list. For example, &quot;Pass, Fail&quot; or &quot;A&quot;, &quot;B&quot;, &quot;C&quot;, &quot;D&quot;, F&quot;.</inlineHelpText>
```
Comment (MatthewBlanski) [inline force-app/main/default/objects/Test_Score_Definition__mdt/fields/Possible_Results__c.field-meta.xml:7]: Likewise here for wrapping every value in quotes.
```suggestion
    <inlineHelpText>For tests that use non-numerical scores, the possible results expressed as a comma-delimited list. For example, &quot;Pass&quot;, &quot;Fail&quot; or &quot;A&quot;, &quot;B&quot;, &quot;C&quot;, &quot;D&quot;, F&quot;.</inlineHelpText>
```

**Review by MatthewBlanski — CHANGES_REQUESTED** [review]
I am requesting a few minor changes to make the descriptions a little more accurate and open up field visibility.

File: force-app/main/default/objects/Test_Definition__mdt/fields/Academic_Level__c.field-meta.xml (hunk 3 above)
Comment (xinnanlisf) [reply force-app/main/default/objects/Test_Definition__mdt/fields/Academic_Level__c.field-meta.xml:6]: just confirming that we are ok with not push upgradable?

Comment (MatthewBlanski) [reply force-app/main/default/objects/Test_Definition__mdt/fields/Academic_Level__c.field-meta.xml:6]: These are represented as data and would have potentially been custom objects in a different world. So yes.

**Comment by xinnanlisf** [comment]
@MatthewBlanski should be good for another review. thanks!

**Review by MatthewBlanski — APPROVED** [review]

**Pull request merged**